### DISCUSSION TITLE
Hubitat automatically stripping <br> from HTML Tile Templates preference entry

### DISCUSSION THREAD

**Discussion by Beelzibub187** [sburke781/ecowitt#8] [Q&A]
I am trying to customize an HTML tile for my air quality sensor, but having issues with manually entering it into the Preferences section rather than using the templates. When I do so and hit save, it just ends up stripping out any <br> entries I entered in the string. Am I stuck with only using the templates? If so, can I somehow make my own template?

Here is the string I am entering:

`Current: ${carbonDioxide} PPM<br>24 hr Average: ${carbonDioxide_avg_24h} PPM`

Here is what it turns into after hitting save on the preferences:

`Current: ${carbonDioxide} PPM24 hr Average: ${carbonDioxide_avg_24h} PPM`

I am pretty new at customizing my Hubitat, so please be easy on me as I'm likely doing something wrong!

**Comment by sburke781** [comment]
Hi @Beelzibub187,

Apologies for not noticing this earlier, I'd like it if GitHub put an entry in it's notifications section to let me know someone is trying to get in touch....  This is the second example of this in recent times.... I will look at settings options to see if I get notified of these kinds of posts...

In terms of the br tags being stripped out... Some time ago now Hubitat introduced limitations on what text could be included in certain parts of the HE platform, in particular HTML tags.  I could make a guess at the reasons why, but I am not 100% sure, and never had a pressing need to understand these.  I believe they are quite firm in their decision in this space, so I don't expect this to change any time soon.

I've only dabbled in a minor way with the templates setup by the original developer, Mirco, but can take a look at whether I may be able to add something like you have tried to setup yourself.  If that doesn't work for any reason, I'm sure we can come up with something using a bit of CSS magic on your dashboard.

I have a pretty bad habit of sticking to deadlines in this space, so don't be afraid to hassle me.... :-)  But this is hopefully a relatively simple one, so you never know...
